src/UBAFeeCalculator/UBAFeeUtility.ts

```diff
@@ -72,7 +78,7 @@ export function getInterval(
 ): [number, [BigNumber, BigNumber]] {
   let result: [number, [BigNumber, BigNumber]] = [
     -1,
-    [BigNumber.from(-MAX_SAFE_JS_INT), BigNumber.from(MAX_SAFE_JS_INT)],
+    [BigNumber.from(-MAX_SAFE_JS_INT).mul(parseEther("1.0")), BigNumber.from(MAX_SAFE_JS_INT).mul(parseEther("1.0"))],
```

Comment:
nit: do we store `parseEther("1.0")` anywhere as a constant? Feel like its a very commonly used value. But i can't recall if its in `relayer-v2`/`sdk-v2`/`contracts-v2`. I feel like @pxrl has put this somewhere nice

Reply:
It's defined in the sdk - good call.